slider/beatmap.py

```diff
@@ -909,6 +909,43 @@ def _get_as_bool(groups, section, field, default=no_default):
         )
 
 
+def _moving_average_by_time(data, delta, num):
```

Comment:
If we split the time column out as I discussed below, we will want to pass it as a separate argument here.